compose-upgrade/entrypoint.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh -l
+set -eux
+
+# Set deploy key
+KEY_PATH=/tmp/deploy_key
+echo "$INPUT_PRIVATE_KEY" > $KEY_PATH
+sed -i 's/\\n/\n/g' $KEY_PATH
+chmod 600 $KEY_PATH
+
+# upgrade
+echo "🚧 Start to upgrade"
+
+# Backup remote host docker-compose file
+ssh $INPUT_USER@$INPUT_HOST -p $INPUT_PORT -i $KEY_PATH -o StrictHostKeyChecking=no -T \
+	"cp $INPUT_DOCKER_COMPOSE_FILE_PATH $INPUT_DOCKER_COMPOSE_FILE_PATH.bak"
```

Comment:
给一个时间戳在这里吧（后面可以考虑把是否备份做成一个参数来控制，比如指定一个备份文件夹）
```suggestion
	"cp $INPUT_DOCKER_COMPOSE_FILE_PATH $INPUT_DOCKER_COMPOSE_FILE_PATH.${date}.bak"
```

Reply:
好的，可以的